src/server.cpp

```diff
@@ -740,7 +740,7 @@ void CServer::SetEnableRecording ( bool bNewEnableRecording )
 
 #if QT_VERSION >= QT_VERSION_CHECK(5, 5, 0)
 // TODO we should use the ConsoleWriterFactory() instead of qInfo()
-        qInfo() << "Recording state " << ( bEnableRecording ? "enabled" : "disabled" );
+        qInfo() << "Recording state" << ( bEnableRecording ? "enabled" : "disabled" );
```

Comment:
Why did you remove the space here?

Reply:
Because qInfo() adds a space.

Reply:
Oh, I did not know that. Thanks for the info.